platform-ui-skin/src/main/webapp/skin/less/platform/skin/uiSimpleContainers/responsiveColumnGroupContainer.less

```
@@ -0,0 +1,10 @@
+.UIContainer.UIResponsiveColumnGroupContainer .UIRowContainer.flexColumnContainer {
+    display: flex;
+    flex-direction: row;
+}
```

```suggestion
.UIContainer.UIResponsiveColumnGroupContainer {
  .UIRowContainer.flexColumnContainer  {
    display: flex;
    flex-direction: row;
   } 
}
```

please format all less files